src/neural_data_simulator/decoder/run_decoder.py

```diff
@@ -17,32 +18,47 @@
 from neural_data_simulator.decoder.settings import DecoderSettings
 from neural_data_simulator.util.runtime import configure_logger
 from neural_data_simulator.util.runtime import get_abs_path
+from neural_data_simulator.util.runtime import get_configs_dir
 from neural_data_simulator.util.runtime import initialize_logger
 from neural_data_simulator.util.runtime import open_connection
-from neural_data_simulator.util.settings_loader import get_script_settings
+from neural_data_simulator.util.settings_loader import check_config_override_str
+from neural_data_simulator.util.settings_loader import load_settings
 
 SCRIPT_NAME = "nds-decoder"
 logger = logging.getLogger(__name__)
 
 
-class _Settings(VersionedYamlModel):
+class _Settings(BaseModel):
     """Decoder app settings."""
 
     log_level: LogLevel
     decoder: DecoderSettings
     timer: TimerModel
 
 
-def _parse_args_settings_path() -> Path:
-    """Parse command-line arguments for the settings path."""
-    parser = argparse.ArgumentParser(description="Run decoder.")
+def _parse_args():
+    parser = argparse.ArgumentParser(
+        description="Decode behavior from input neural data.",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
     parser.add_argument(
         "--settings-path",
         type=Path,
+        default=Path(get_configs_dir()).joinpath("settings_decoder.yaml"),
         help="Path to the settings_decoder.yaml file.",
     )
+    parser.add_argument(
+        "--overrides",
+        "-o",
+        nargs="*",
+        type=check_config_override_str,
+        help=(
+            "Settings file overrides, e.g. "
+            "'log_level=DEBUG decoder.spike_threshold=-210'."
```

Comment:
an error occurs if I run the decoder with the suggested override:
```
decoder -o 'log_level=DEBUG decoder.spike_threshold=-210'     
Args ->  Namespace(settings_path=PosixPath('/Users/nsander/.nds/settings_decoder.yaml'), overrides=['log_level=DEBUG decoder.spike_threshold=-210'])
Traceback (most recent call last):
  File "/Users/nsander/.pyenv/versions/3.11.6/bin/decoder", line 6, in <module>
    sys.exit(run())
             ^^^^^
  File "/Users/nsander/workspace/neural-data-simulator/src/neural_data_simulator/decoder/run_decoder.py", line 88, in run
    load_settings(
  File "/Users/nsander/workspace/neural-data-simulator/src/neural_data_simulator/util/settings_loader.py", line 55, in load_settings
    settings = settings_parser.parse_obj(settings_dict)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "pydantic/main.py", line 527, in pydantic.main.BaseModel.parse_obj
  File "pydantic/main.py", line 342, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for _Settings
log_level
  value is not a valid enumeration member; permitted: 'DEBUG', 'INFO', 'ERROR', 'WARNING', 'CRITICAL' (type=type_error.enum; enum_values=[<LogLevel._DEBUG: 'DEBUG'>, <LogLevel._INFO: 'INFO'>, <LogLevel._ERROR: 'ERROR'>, <LogLevel._WARNING: 'WARNING'>, <LogLevel._CRITICAL: 'CRITICAL'>])
```

If I pass a single override it works:
```
decoder -o 'log_level=DEBUG'                             
Args ->  Namespace(settings_path=PosixPath('/Users/nsander/.nds/settings_decoder.yaml'), overrides=['log_level=DEBUG'])
DEBUG [nds-decoder]: run_kecoder settings:
decoder:
```

Reply:
Ah, I miscommunciated the usage in the `help` description. The new help message is:
```
  --overrides [OVERRIDES ...], -o [OVERRIDES ...]
                        Specify settings overrides as key-value pairs, separated by spaces. For example: -o log_level=DEBUG streamer.lsl_chunk_frequency=50 (default: None)
```
The `''` quotes were meant to indicate "these are the arguments," but I see now that was misleading. Hope the updated help-message is clearer!